src/components/Vault/BasicVault/CardContent.tsx

```diff
@@ -5,7 +5,7 @@ import BigNumber from 'bignumber.js';
 import { useWeb3Account } from 'clients/web3';
 import NumberFormat from 'react-number-format';
 import { commaFormat } from 'utilities/common';
-import { TokenSymbol } from 'types';
+import { TokenId } from 'types';
```

Comment:
nice. I like the change because it clarifies what property we are using. Per the comment in Slack I went with symbol for what we display. We could add name back if we every display the long name of tokens but we currently don't